ElectronNET.API/Clipboard.cs

```diff
@@ -170,5 +177,12 @@ public void WriteImage(NativeImage image, string type = "")
         {
             BridgeConnector.Emit("clipboard-writeImage", JsonConvert.SerializeObject(image), type);
         }
+
+        private static readonly JsonSerializer _jsonSerializer = new()
```

Comment:
This pattern of serializer is used so pervasively in the project that it would be beneficial just to have have a global static one defined and referenced.